src/utils/subscriberBreaches.ts

```diff
@@ -0,0 +1,124 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getUserEmails } from "../db/tables/emailAddresses.js";
+import { HibpLikeDbBreach, getBreachesForEmail } from "./hibp.js";
+import { getSha1 } from "./fxa.js";
+import { BreachDataTypes, filterBreachDataTypes } from "./breachResolution.js";
+import {
+  Breach,
+  Subscriber,
+} from "../app/(nextjs_migration)/(authenticated)/user/breaches/breaches.js";
+
+export interface SubscriberBreach {
+  addedDate: string;
+  breachDate: string;
+  dataClasses: Array<string>;
+  description: string;
+  domain: string;
+  id: number;
+  isResolved?: boolean;
+  favIconUrl: string;
+  modifiedDate: string;
+  name: string;
+  title: string;
+  emailsEffected: string[];
```

Comment:
Not sure if we have a style guide, but should this be `string[]` or `Array<string>` like on L17 above (and L28 below)?

Reply:
We do have it both ways, but I think we usually go with string[] so I've changed the other one. Thanks